### DISCUSSION TITLE
How to install BNB Greenfield CLI in linux systems?

### DISCUSSION THREAD

**Discussion by sudharsansubash2002** [bnb-chain/greenfield#550] [Q&A]
Hi Everyone , I have been trying to install greenfield cli in Linux aws ec2 machine. But Getting the following error when calling "make build" command : 

> ubuntu@ip-172-31-35-45:~/greenfield-cmd$ make build
go build -o ./build/gnfd-cmd cmd/*.go
# internal/abi
../go/src/internal/abi/funcpc.go:22:6: FuncPCABI0 redeclared in this block
        ../go/src/internal/abi/abi.go:117:6: other declaration of FuncPCABI0
../go/src/internal/abi/funcpc.go:31:6: FuncPCABIInternal redeclared in this block
        ../go/src/internal/abi/abi.go:126:6: other declaration of FuncPCABIInternal
# github.com/herumi/bls-eth-go-binary/bls
../go/pkg/mod/github.com/herumi/bls-eth-go-binary@v0.0.0-20210917013441-d37c07cfda4e/bls/cast.go:9:28: undefined: SecretKey
../go/pkg/mod/github.com/herumi/bls-eth-go-binary@v0.0.0-20210917013441-d37c07cfda4e/bls/cast.go:9:40: undefined: Fr
../go/pkg/mod/github.com/herumi/bls-eth-go-binary@v0.0.0-20210917013441-d37c07cfda4e/bls/cast.go:13:26: undefined: Fr
../go/pkg/mod/github.com/herumi/bls-eth-go-binary@v0.0.0-20210917013441-d37c07cfda4e/bls/cast.go:13:31: undefined: SecretKey
../go/pkg/mod/github.com/herumi/bls-eth-go-binary@v0.0.0-20210917013441-d37c07cfda4e/bls/cast.go:19:28: undefined: PublicKey
../go/pkg/mod/github.com/herumi/bls-eth-go-binary@v0.0.0-20210917013441-d37c07cfda4e/bls/cast.go:19:40: undefined: G1
../go/pkg/mod/github.com/herumi/bls-eth-go-binary@v0.0.0-20210917013441-d37c07cfda4e/bls/cast.go:23:26: undefined: G1
../go/pkg/mod/github.com/herumi/bls-eth-go-binary@v0.0.0-20210917013441-d37c07cfda4e/bls/cast.go:23:31: undefined: PublicKey
../go/pkg/mod/github.com/herumi/bls-eth-go-binary@v0.0.0-20210917013441-d37c07cfda4e/bls/cast.go:29:23: undefined: Sign
../go/pkg/mod/github.com/herumi/bls-eth-go-binary@v0.0.0-20210917013441-d37c07cfda4e/bls/cast.go:29:30: undefined: G2
../go/pkg/mod/github.com/herumi/bls-eth-go-binary@v0.0.0-20210917013441-d37c07cfda4e/bls/cast.go:29:30: too many errors
make: *** [Makefile:6: build] Error 1

Is there any solution for this?

**Comment by Firas2515** [comment]
```
``
```

**Reply by Firas2515** [reply]
```
````
``
````
```
